hack/setup-test-cluster.sh

```diff
@@ -22,7 +22,7 @@ kind: MachineConfigPool
 metadata:
   name: worker-cnf
   labels:
-    worker-cnf: ""
+    worker-rt: ""
```

Comment:
shouldn't this be worker-cnf?

Reply:
functionally, this might not matter at all, since the node selector is worker-cnf still, but it looks odd

Reply:
@davidvossel it matters!

@fedepaol sorry, this was miscommunication and bad copy and paste of me in our chat: I did not mean to use `worker-rt` instead of `worker-cnf`, but to use the machineconfig domain in the label. So the complete label should be `machineconfiguration.openshift.io/role: worker-cnf`

And the reason why this matters: the operator uses the (long version) of this label as MCP selector in the KubeletConfig it creates. CI didn't catch this yet because the performance tests are not merged yet.

Reply:
Ah, thanks for explaining. Will fix it in the tests PR.